src/Accordion/Accordion.js

```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled';
+
+import Box from '../Box';
+
+const Accordion = styled(Box)`
+  border: 1px solid ${props => props.theme.colors.border.default};
+  border-radius: ${props => props.theme.borderRadius.large};
```

Comment:
It was probably done in a later PR, but I believe you added styled-system border and border radius. I wonder if we should use that as default props instead? May need to be outside of this PR. Just a thought.

Reply:
You are correct. This is was done in the next PR to support the new header component.